config.xml

```diff
@@ -1,70 +1,73 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
What caused these double quotes to change to single quotes, and dropped the `standalone` attribute?

We want the code in this repo to match whatever would be generated on `ionic build`, don't you think?